app/grandchallenge/algorithms/emails.py

```diff
@@ -0,0 +1,70 @@
+from django.contrib.sites.models import Site
+
+from grandchallenge.core.utils.email import send_templated_email
+
+
+def send_permission_request_email(obj):
+    """
+    Email the editors that someone has requested to view an algorithm.
+
+    obj:         AlgorithmPermissionRequest object containing info on which
+                 user requested access to which algorithm.
+    """
+    title = f"[{obj.algorithm.title}] New access request"
+    kwargs = {
+        "user": obj.user,
+        "site": Site.objects.get_current(),
+        "algorithm": obj.algorithm,
+    }
+    for editor in obj.algorithm.editors_group.user_set.all():
+        kwargs["editor"] = editor
+        send_templated_email(
+            title,
+            "algorithms/emails/access_request.html",
+            kwargs,
+            [editor.email],
+        )
+
+
+def send_permission_granted_email(obj):
+    """
+    Email the requester that their request has been approved.
+
+    obj:         AlgorithmPermissionRequest object containing info on which
+                 user requested access to which algorithm.
```

Comment:
Could you set your IDE to generate numpy style docstrings? Thanks!

Reply:
Sure, updated :slightly_smiling_face: